File: Java/src/main/java/com/hillnet/AddTwoDigits.java
package com.hillnet;

public class AddTwoDigits {

	public int sumTwoDigits(int j) {
		
		
        int sum = 0;
        while (j > 0) {
            sum = sum + j % 10;
            j = j / 10;
        }
        return sum;
	}
}

File: cpp/addTwoDigits.cpp
#include <iostream>

int addTwoDigits(int n) {
    int k = n % 10;
    int x = (n-k)/10;
    std::cout << x+k;

    return 0;
}

int main()
{
    addTwoDigits(50);
    return 0;
}
